moncli/entities/item.py

```diff
@@ -618,6 +618,59 @@ def get_column_value(self, id = None, title = None, *args):
 
         return self.column_values[id]
     
+    def change_column_title(self,column, title: str, *args):
```

Comment:
It was not specified in the AC to add this functionality to the Item entity.  Only boards were specified because Columns are an entity related only to Boards.  Column Values are thus related to Items, but Column Values are not Columns and thus required different behavior.

Please remove this code.